### PR TITLE
GraphQL - add media streams

### PR DESCRIPTION
**Pull request by toyhammered**

### CODE REVIEW

**Comment by toyhammered** [comment]
Need to add a new regions field and then also have a dubs/subs field (we will keep the video sub_language and dub_language, but deprecate) and then just add subs/dubs to them, and these will just be an array.

**Review by timw4mail — APPROVED** [review]

File: app/graphql/types/anime.rb [hunk 1]
```diff
@@ -11,4 +11,12 @@ class Types::Anime < Types::BaseObject
     null: true,
     method: :youtube_video_id,
     description: 'Video id for a trailer on YouTube'
+
+  field :streaming_links, Types::StreamingLink.connection_type,
+    null: false,
+    description: ''
```
Comment (NuckChorris) [inline app/graphql/types/anime.rb]: Could you add a description for this?

File: app/graphql/types/video.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class Types::Video < Types::BaseObject
+  implements Types::Interface::Streamable
+
+  description ''
```
Comment (NuckChorris) [inline app/graphql/types/video.rb]: you're missing a bunch of descriptions

**Review by NuckChorris — CHANGES_REQUESTED** [review]

File: app/graphql/types/video.rb (hunk 2 above)
Comment (toyhammered) [reply app/graphql/types/video.rb]: I shoulda converted to draft instead of WIP, sorry.

File: app/graphql/types/video.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    null: false,
+    description: 'The language this is subbed in',
+    deprecation_reason: 'Please use subs',
+    method: :sub_lang
+
+  field :dub_language, String,
+    null: false,
+    description: 'The language this is dubbed in.',
+    deprecation_reason: 'Please use dubs',
+    method: :dub_lang
```
Comment (NuckChorris) [inline app/graphql/types/video.rb]: Please don't add new fields that are deprecated from day one, just leave em off

**Review by NuckChorris — CHANGES_REQUESTED** [review]

**Review by NuckChorris — APPROVED** [review]

**Pull request merged**